src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Options/PackageSourceValidator.cs

```diff
@@ -117,6 +117,7 @@ internal static void EnsureValidSources(PackageSource packageSource)
             }
         }
 
+        // Currently not used. Follow-up in https://github.com/NuGet/Home/issues/14507.
```

Comment:
To avoid having dead code, consider deleting the unused methods and rely on source control for when we need to bring it back.

Reply:
That's correct! The follow-up issue is tracking to either remove the dead code, or to reimplement it, depending on how Unified Settings decides to address the issue.